### PR TITLE
Enhancing user experience for humaneval benchmark

### PR DESCRIPTION
**Pull request by bhanvimenghani**
This pr adds a Cleanup script, a Run script, enables user to provide input in terms of duration for how long it wants to keep the workload running. This has been tested on the nerc cluster. 
This is how the user experience would look like!
image

### CODE REVIEW

File: human-eval-benchmark/Readme.md [hunk 1]
```diff
@@ -21,6 +21,11 @@ Towards the end of the script B the user is prompted to fill the run duration, w
 
 ## 2. Automated Job
 
-In this approach we already have a combined script named `script.py`, and a Docker file which is used to create this docker image `quay.io/kruizehub/human-eval-deployment` which is used in the `job.yaml` file.
+To run the benchmark in an automated way the user simply needs to login to the relevent Openshift AI cluster, create a namespace or you can use the default namespace. Set your desired environment variable in `job.yaml`, you have number of prompts or duration to choose from. If num_prompts and duration_in_seconds both are set num_prompts has a higher precedence. Apply `pcv.yaml` followed by applying `job.yaml`. This would deploy the humaneval benchamrk in the specified namespace.
```
Comment (kusumachalasani) [inline human-eval-benchmark/Readme.md:24]: Typo for pvc.yaml

File: human-eval-benchmark/run_humaneval.sh [hunk 2]
```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+JOB_YAML=./manifests/job.yaml
+JOB_NAME=human-eval-deployment-job
+NAMESPACE=default
```
Comment (kusumachalasani) [inline human-eval-benchmark/run_humaneval.sh:5]: Can you provide an option to change the namespace as parameter. By default, the namespace can be 'default'

Comment (bhanvimenghani) [reply human-eval-benchmark/run_humaneval.sh]: @kusumachalasani added it!

**Review by kusumachalasani — APPROVED** [review]
lgtm!

**Pull request merged**